data/json/regional_map_settings.json

```diff
@@ -267,6 +268,7 @@
             "clear_types": false,
             "types": {
               "t_underbrush": 30,
+              "t_fern": 15,
```

Comment:
I'll have to try this out, but this number seems too high. Do ferns really make up 1/3 of the ground cover in a New England forest?

Reply:
I'm happy to change it to whatever feels the best for gameplay.
According to (Penn State)[https://ecosystems.psu.edu/research/centers/private-forests/news/2016/new-forest-science-fact-sheet-on-fern-impacts-and-control]

>The latest U.S. Forest Service forest inventory and analysis data indicate that rhizomatous ferns species, hay-scented, New York, and bracken fern, now comprise over 20% of Pennsylvania’s forest understories.

Reply:
I added the mod and took out the looks-like field. Here's a picture of a forest at max zoom:
![image](https://user-images.githubusercontent.com/16219776/65848896-ac852e00-e30d-11e9-92dc-fe000b6d0c24.png)
1 zoom level in from max:
![image](https://user-images.githubusercontent.com/16219776/65848913-c9b9fc80-e30d-11e9-8e6d-d8724d2f38bf.png)
Thinning them out a bit seems appropriate. I'll reduce my numbers to 9 and come back with a picture.